doc/content/examples/example3.md

```diff
@@ -87,4 +87,36 @@ DEEPSOIL and presented in
 Further details regarding to application of MASTODON to simulate seismic site response and soil
 structure interaction analysis can be found in [!citet](baltaji2017nonlinear).
 
+## Implementation using Automatic Differentiation
+
+This example can also be run using Automatic Differentiation (AD). Automatic Differentiation
+computes the numerical Jacobian that is very close to the theoretical Jacobian.  
+More information can be found in [ISoil material](source/materials/ADComputeISoilStress.md).
+
+The following two additions to the input file should be made to run this example using AD.
+Add 'use_automatic_differentiation = true' in 'Kernels/DynamicTensorMechanics':
+
+!listing examples/ex03/shear_beam_Isoil_free_field_AD.i
+         start=Kernels
+         end=AuxKernels
+
+Also, add 'use_automatic_differentiation = true' in 'Materials/I_Soil/soil_all':
+
+!listing examples/ex03/shear_beam_Isoil_free_field_AD.i
+         start=Materials
+         end=Preconditioning
+
+The plots below present the accelerations and response spectra at the top soil computed
```

Comment:
In the plots below, please change 'top soil accelerations' to 'accelerations at the soil surface'. Also, switch on the grid in the plots.

Reply:
@somu15 please change the captions as well.